android-gif-drawable/src/main/java/pl/droidsonroids/gif/GifIOException.kt

```diff
@@ -16,7 +16,7 @@ class GifIOException internal constructor(errorCode: Int, private val mErrnoMess
     override val message: String
         get() = if (mErrnoMessage == null) {
             reason.formattedDescription
-        } else reason.formattedDescription + ": " + mErrnoMessage
+        } else "${reason.formattedDescription} : $mErrnoMessage"
```

Comment:
tests fail due inconsistent messages
```suggestion
        } else "${reason.formattedDescription}: $mErrnoMessage"
```

Reply:
It is so weird. It did not fail in my local system